Gruntfile.js

```diff
@@ -94,6 +94,7 @@ grunt.initConfig({
 		qunit: [
 			"test/index.html",
 			"test/autostart.html",
+			"test/amd.html",
```

Comment:
@leobalter pointed out that I should add this to the Browserstack tests as well. Is there a reason the two lists aren't in sync? I found this list by searching for "autostart.html".